packages/web-app-files/src/components/Search/List.vue

```diff
@@ -278,57 +289,71 @@ export default defineComponent({
     )
 
     const buildSearchTerm = (manuallyUpdateFilterChip = false) => {
-      let query = ''
-      const add = (k: string, v: string) => {
-        query = (query + ` ${k}:${v}`).trimStart()
-      }
+      const q = {}
 
       const humanSearchTerm = unref(searchTerm)
       const isContentOnlySearch = queryItemAsString(unref(fullTextParam)) == 'true'
 
       if (isContentOnlySearch && !!humanSearchTerm) {
-        add('content', `"${humanSearchTerm}"`)
+        q['content'] = `"${humanSearchTerm}"`
       } else if (!!humanSearchTerm) {
-        add('name', `"*${humanSearchTerm}*"`)
+        q['name'] = `"*${humanSearchTerm}*"`
       }
 
       const humanScopeQuery = unref(scopeQuery)
       const isScopedSearch = unref(doUseScope) === 'true'
       if (isScopedSearch && humanScopeQuery) {
-        add('scope', `${humanScopeQuery}`)
+        q['scope'] = `${humanScopeQuery}`
       }
 
-      const humanTagsParams = queryItemAsString(unref(tagParam))
-      if (humanTagsParams) {
-        add('tag', `"${humanTagsParams}"`)
-
-        if (manuallyUpdateFilterChip && unref(tagFilter)) {
+      const updateFilter = (v: Ref) => {
+        if (manuallyUpdateFilterChip && unref(v)) {
```

Comment:
since this is kind of nested, I'd love to have more decriptive variable names then k,v,q

Reply:
@JanAckermann can you please change the naming, I’m not at my desk anymore and have vacation for the rest of the week